types/bull-arena/index.d.ts

```diff
@@ -1,56 +1,61 @@
-// Type definitions for bull-arena 2.6
+// Type definitions for bull-arena 3.0
 // Project: https://github.com/bee-queue/arena/
 // Definitions by: Levi Bostian <https://github.com/levibostian>
+//                 Gaurav Sharma <https://github.com/gtpan77>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 4.1
 
 import { RequestHandler } from "express";
 import { ClientOpts } from "redis";
+import Bull = require("bull");
+import Bee = require("bee-queue");
```

Comment:
What is the reason for writing the import statement in this format instead of `import {...} from "...";` like we are doing above?

Reply:
I did use import but npm test throw error given below and suggested to use require. Require is also used by other @types modules.
```
ERROR: 10:8   no-import-default-of-export-equals  The module "bull" uses `export = `. Import with `import Bull = require("bull")`.
```

Reply:
Ok